gitnamehere/point-of-sale-group-project
# Business Information and POS home page overhaul

This PR allows the business to customize their business information (and have it reflect on the frontend) and overhauls the POS home page.

Business information customization is done via a script and ids
Simply add in the script tag such as the one in the bottom of `public/store/index.html` and set the corresponding element ids to dynamically update the text to business info

(I did not feel like doing this via ejs)

Store menu and ordering page can be updated once they are being worked on (combine them together)

Testing:
* `npm run setup-demo:dev`
* `npm run start:dev`
* log in to POS
* test out the buttons, and verify they work (except for the account which looks like a button)
* go to the settings page
* update the business info
* go to the store home page
* verify the changes are reflected there and in the POS home page
